src/libpisp/frontend/frontend.cpp

```diff
@@ -398,8 +398,7 @@ void FrontEnd::Prepare(pisp_fe_config *config)
 		if (dirty_flags & block_enable(PISP_FE_ENABLE_OUTPUT0, i))
 		{
 			pisp_image_format_config &image_config = fe_config_.ch[i].output.format;
-
-			getOutputSize(i, image_config.width, image_config.height);
+			fixOutputSize(i);
```

Comment:
Micro-nitpick: we have a blank line at L388 but no blank line here. Come to think of it, do we still need the declaration?